fix(icongenie): Deprecated media feature (fix #8118)

Blfrg opened the pull request:
<!--
Please make sure to read the Pull Request Guidelines:
https://github.com/quasarframework/quasar/blob/dev/.github/CONTRIBUTING.md#pull-request-guidelines
-->

<!-- PULL REQUEST TEMPLATE -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [x] Documentation
- [x] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**
Closes #8118

Although [the spec](https://www.w3.org/TR/mediaqueries-4/#mf-deprecated) is still technically a [CR](https://www.w3.org/2020/Process-20200915/#RecsCR), rather than an [REC](https://www.w3.org/2020/Process-20200915/#RecsW3C),
universal implementation by the browsers suggests it will maintain reliability for the foreseeable future.

This change would encourage the use of the non-deprecated @&#8203;media features '[height](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/height)' and '[width](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/width)'
to write CSS that passes [w3 validation](https://validator.w3.org/).

Blfrg committed:
fix(icongenie): Deprecated media feature

device-width -> width
device-height -> height
https://www.w3.org/TR/mediaqueries-4/#mf-deprecated

Blfrg committed:
docs(icongenie): Deprecated media feature

device-width -> width
device-height -> height
https://www.w3.org/TR/mediaqueries-4/#mf-deprecated

rstoenescu commented:
Good catch! Thanks for the PR! Will merge it asap and release a new icongenie.

hawkeye64 commented:
@rstoenescu Just a ping, so you don't forget. 
Ref: https://github.com/quasarframework/quasar/issues/8118

rstoenescu commented:
A quick follow-up. Just want to be sure that we are on the same page. Have you tested the PWA with only the tag corresponding to your device's size? The fact that the format is obsolete doesn't makes me confident that iOS is compliant with the newer w3c specs..

Blfrg commented:
@rstoenescu Sorry for the delay in my response;
I don't actually have any iOS devices at my disposal.

I tried to test on [BrowserStack](https://browserstack.com), but they only support "Add to home" on v9 and lower.
My sample code didn't parse correctly on the older device, and we're testing a recent browser change anyway.

I was able to finally get ahold of an "iPhone 11 Pro Max" which is said to meet these display parameters:
"viewport size 414 x 896 Pixels and its pixel ratio is about 3" = 1242x2688 pixels

Corresponding to this **original** link element:
```html
    <!-- iPhone XS Max -->
    <link
      rel="apple-touch-startup-image"
      media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 3)"
      href="icons/apple-launch-1242x2688.png"
    />
```

And this **updated** link element:
```html
    <!-- iPhone XS Max -->
    <link
      rel="apple-touch-startup-image"
      media="(width: 414px) and (height: 896px) and (-webkit-device-pixel-ratio: 3)"
      href="icons/apple-launch-1242x2688.png"
    />
```

I tested with **only** the relevant [size] link element as you suggested.
Both original and updated versions just displayed a black screen very briefly.
I reverted to the full list of screen sizes, tested original and updated separately,
and these both displayed a black screen briefly as well.

Maybe the app loaded too quickly for a screen to be displayed,
but even under original full list of screen and image sizes
I was not able to actually see any startup screen.

Note: on Android I do see a proper startup screen.

Maybe someone else can test to confirm?
